hugegraph-llm/src/hugegraph_llm/config/models/base_prompt_config.py

```
@@ -60,25 +61,51 @@ def ensure_yaml_file_exists(self):
             "\n".join([f"    {line}" for line in self.custom_rerank_info.splitlines()])
         )
+        
+        indented_gremlin_prompt = "\n".join([f"  {line}" for line in self.gremlin_generate_prompt.splitlines()])
+        indented_example_prompt = "\n".join([f"    {line}" for line in self.extract_graph_prompt.splitlines()])
         indented_default_answer_template = "\n".join([f"    {line}" for line in self.answer_prompt.splitlines()])
         indented_keywords_extract_template = (
             "\n".join([f"    {line}" for line in self.keywords_extract_prompt.splitlines()])
         )
         indented_doc_input_text = "\n".join([f"  {line}" for line in self.doc_input_text.splitlines()])
 
+        indented_language_generated = "\n".join([f"  {line}" for line in self.llm_settings.language.splitlines()])
```

Pylint (3.3.7)

[error] Instance of 'BasePromptConfig' has no 'llm_settings' member

(E1101)